### ISSUE TITLE
[Screen Reader – Make Code – Project Setting] Screen reader announces extra information while navigating 'Toggle , Save & Edit setting as text' buttons under 'Project setting' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3686]
**User Experience:**
User would be confused if screen reader announces irrelevant information for any control which doesn't relevant to that control and also he/she won't get the context of such irrelevant information. Not only this, they will proceed further and will miss the functionality.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Project Settings" option and select it.
7. Navigate though all the controls appearing after select "Project Setting" option i.e. No Pairing Required: Anyone can connect via Bluetooth, Passkey pairing: Pairing requires 6 digit key to pair etc. 
8. Verify that Screen reader announces extra information while navigating 'Toggle , Save & Edit setting as text' buttons or not.

**Actual Result:**
Screen reader announces extra information as 'Type a name for your project edit testing' while navigating 'Toggle , Save & Edit setting as text' buttons under 'Project setting' screen.

**Expected Result:**
Screen reader should not announce extra information as 'Type a name for your project edit testing' while navigating 'Toggle , Save & Edit setting as text' buttons under 'Project setting' screen. It should only provide focused control information.

**Note:**
This issue does not exists with Firefox and Chrome Browser.

**WCAG Reference:**
[WCAG1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.3.1; #Win10-Edge(Chromium)

[WCAG1.3.1_Project Setting_Extra information.zip](https://github.com/microsoft/pxt-microbit/files/5534940/WCAG1.3.1_Project.Setting_Extra.information.zip)

**Comment by jwunderl** [comment]
Same as https://github.com/microsoft/pxt-microbit/issues/3688; browser is deciding how to give the information to the user, in this case giving an overview then more detailed information of the individual portions.

In this case it seems like potentially a narrator bug that it is showing the narrator focus on the wrong portion of the screen -- if you tab through or go backwards, the box shows up over the correct elements. This focus is not ours though / we do not control it, so closing as external.
